backends/platform/sdl/opendingux/build_miyoomini.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+TOOLCHAIN=/opt/miyoomini-toolchain
+SYSROOT=$TOOLCHAIN/arm-linux-gnueabihf
+
+export PATH=$TOOLCHAIN/usr/bin:$SYSROOT/usr/include:$TOOLCHAIN/bin:$PATH
+export CXX=arm-linux-gnueabihf-g++
+
+./configure --host=miyoomini --enable-release --disable-detection-full --enable-plugins --default-dynamic --enable-engine=testbed
+
+make -j5 all sd-root sd-zip
```

Comment:
The build scripts should be in a better place in backends/platform/sdl/miyoo

The devices are not part of opendingux, they use arm cpus and opendingux is mips only

Reply:
Done